tests/common/cached_transformers_test.py

```diff
@@ -14,6 +19,44 @@ def test_get_missing_from_cache_local_files_only(self):
                 local_files_only=True,
             )
 
+    def test_from_pretrained_avoids_weights_download_if_override_weights(self):
+        # only download config because downloading pretrained weights in addition takes too long
+        transformer = AutoModel.from_config(
+            AutoConfig.from_pretrained("bert-base-uncased", cache_dir=self.TEST_DIR)
```

Comment:
Let's use `epwalsh/bert-xsmall-dummy` instead:

```suggestion
            AutoConfig.from_pretrained("epwalsh/bert-xsmall-dummy", cache_dir=self.TEST_DIR)
```